src/DonationForms/Migrations/CleanMultipleSlashesOnDB.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Give\DonationForms\Migrations;
+
+use Give\DonationForms\Actions\SanitizeDonationFormPreviewRequest;
+use Give\DonationForms\Models\DonationForm;
+use Give\DonationForms\Properties\FormSettings;
+use Give\Framework\Blocks\BlockCollection;
+use Give\Framework\Exceptions\Primitives\Exception;
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class CleanMultipleSlashesOnDB extends Migration
+{
+
+    /**
+     * @unreleased
+     *
+     * @throws Exception
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $formMetaTable = $wpdb->prefix . 'give_formmeta';
+
+        $sql = $wpdb->prepare("SELECT form_id FROM $formMetaTable as fm WHERE
+        (fm.meta_key = 'formBuilderSettings' AND fm.meta_value LIKE '%\\\\\\\\\\\\\\\\\\\\\\\\%')
+        OR
+        (fm.meta_key = 'formBuilderFields' AND fm.meta_value LIKE '%\\\\\\\\\\\\\\\\\\\\\\\\%')");
+
+        $formIds = $wpdb->get_results($sql, ARRAY_A);
+
+        foreach ($formIds as $formId) {
+            $form = DonationForm::find((int)$formId['form_id']);
+            $settings = (new SanitizeDonationFormPreviewRequest())($form->settings->toArray());
+            $blocks = (new SanitizeDonationFormPreviewRequest())($form->blocks->toArray());
```

Comment:
Let's pull the functionality of this action directly into the migration.  We always want to limit our production code inside migrations to prevent changes up the chain affecting the migration.

With that in mind, we should attempt to remove the usage of the models here and just try to update the problematic meta by itself so the migration is only touching what it needs to be fixed.

Reply:
Resolved in 85e822c2bfdc2ed2ba18023fab65d29f824b0658